.github/workflows/goreleaser.yml

```diff
@@ -0,0 +1,27 @@
+name: goreleaser
+
+on:
+  push:
```

Comment:
will this flow need those extra token permissions?

Reply:
ah, yes

Reply:
fixed